conf/distro/pelux.conf

```diff
@@ -9,7 +9,7 @@ require conf/distro/poky.conf
 MAINTAINER = "Gordan Markuš <gordan.markus@pelagicore.com>"
```

Comment:
We should remove the maintainer here also.

Reply:
yes, will do in a separate commit

Reply:
actually, I believe this is an obligatory field and yocto-layer-check will fail, but let me check first...

Reply:
nah, it's cool; will remove later.